CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+---
+## [2.0.1] - 2022-09-02
```

Comment:
just notice that you will have to yank `2.0.0` when releasing `2.0.1` as you changed public API and broke semver compatbility